src/SDK/OssClient.cs

```diff
@@ -198,8 +200,16 @@ internal bool IsSdkVersionSetInPackageData(DocumentPackage package)
         public SigningStyleService GetSigningStyleService ()
         {
             return signingStyleService;
-        }       
+        }
 
+        /**
+         * 
+         */
+        public DeclineReasonsService GetDeclineReasonsService()
+        {
+            return declineReasonsService;
+        }
+        
```

Comment:
This service is not required because AccountConfigService will be used. Please delete it.